packages/ember-runtime/lib/ext/string.js

```diff
@@ -27,8 +27,8 @@ if (Ember.EXTEND_PROTOTYPES === true || Ember.EXTEND_PROTOTYPES.String) {
     @for String
     @private
   */
-  StringPrototype.fmt = function () {
-    return fmt(this, arguments);
+  StringPrototype.fmt = function (...args) {
+    return fmt(this, args);
```

Comment:
Ah darn I fell into the trap again. `...arguments` will expand the arguments into `apply`, of course.